### ISSUE TITLE
Add documentation how to develop this package

### ISSUE THREAD

**Issue by tisto** [collective/gatsby-source-plone#7]
So far we made everything work on Travis. Though, the dev environment locally is far from ideal. We have to fix that and document everything to make it easy for newbies and possible GSoC students to jump on.

**Comment by tisto** [comment]
Basic documentation added: https://github.com/collective/gatsby-source-plone/commit/5cc595ebc8a72fb3f0f32303342358fdc8aca6cd

This should be enough to get ppl started.

**Comment by ajayns** [comment]
Right now there's docs on how to get started with development. Since as I work on building the project, I'll be adding docs on its working and use cases etc, what would be the best approach to storing documentation? 

How about a `docs` folder which would contain markdown files for topic-wise documentation? 

Opinions @cekk @tisto  @datakurre

**Comment by tisto** [comment]
Makes sense. If you want to go with Markdown I'd recommend looking into MKDocs. If you want to go with RST, Sphinx is the way to go. 

You can also just start with a simple docs folder to drop
markdown files into. 

Von meinem iPhone gesendet

**Comment by ajayns** [comment]
Okay great, I'll get started with that then. Thanks!

**Comment by datakurre** [comment]
:+1: I keep ReStructuredText superior to Markdown, but Markdown is standard on JavaScript packages.

**Comment by datakurre** [comment]
@tisto @ajayns Isn’t this done with the current README?

**Comment by ajayns** [comment]
Yes I believe it is, I'm unsure if @tisto wanted anything else covered  though?
